pkg/types/health.go

```diff
@@ -49,7 +49,7 @@ type HealthConfig struct {
 func NewHealthConfig() *HealthConfig {
 	return &HealthConfig{
 		SnapshotLeaseRenewalEnabled:     DefaultSnapshotLeaseRenewalEnabled,
-		FullSnapshotLeaseUpdateInterval: wrappers.Duration{Duration: DefaultFullSnapshotLeaseUpdateInterval},
+		FullSnapshotLeaseUpdateInterval: wrappers.Duration{Duration: FullSnapshotLeaseUpdateInterval},
```

Comment:
If now `FullSnapshotLeaseUpdateInterval` is no longer configurable via backup-restore then why it has to be a part of `HealthConfig` ?

Reply:
You're right. I'll remove it from `HealthConfig` and make the `const` accessible directly from `types` pkg. Is this good?

Reply:
yup, that will be better

Reply:
Done. Pls see the latest [commit](https://github.com/gardener/etcd-backup-restore/pull/744/commits/8143434d469c1120126340da638c88a464538b3a)